Code/autopkglib/MunkiImporter.py

```diff
@@ -295,10 +295,11 @@ def main(self):
                 icon_path = icon_library.extract_and_copy_icon_to_repo(
                     self.env["pkg_path"], pkginfo, import_multiple=False
                 )
+
         else:
-            icon_path = ""
+            icon_path = None
 
-        self.env["icon_repo_path"] = icon_path
+        self.env["icon_repo_path"] = icon_path or ""
```

Comment:
Instead of setting this to an empty string, I would consider instead changing the summary display logic to not even include an icon field if no icon was imported. I'm not married to that approach, but I think it's cleaner than having an empty field.

Reply:
This made sense to me too, and I played around with it a bit. Unfortunately, I ran into an issue with the `Code/munki` binary when running more than one recipe at a time. It appears to expect the same `report_fields` availability for each recipe. If recipe A includes `icon_repo_path` in the `report_fields` and corresponding `data` entry, but recipe B does not include either, an error occurs. 

For the time being, I think it is easier to provide the icon column in the output regardless.